packages/components/src/tag/src/Tag.css

```diff
@@ -1,12 +1,11 @@
 .o-ui-tag {
```

Comment:
The Focus / button used in the Tag is problematic

![image](https://github.com/gsoft-inc/wl-orbiter/assets/361632/1e64f742-785a-4944-b2ba-162d7e633fee)
![image](https://github.com/gsoft-inc/wl-orbiter/assets/361632/c1fa4eb1-cec7-410c-a503-0cf75bc71bca)

Reply:
what do you think needs to change for this? The old button size seems to be much smaller, but I didn't change the button size that it's using. Are you comparing with orbit design system? I don't think I have the link to this, Can you send it please?

Reply:
https://orbit.sharegate.design/

Reply:
The issue is due to the fact that tags are now 32px and not 40px. The solution will be to create a buttonSizeAdapter and make sure to use a 2xs button.

Here's an example of how it's been done for the text.

```
const textSize = createSizeAdapter({
    "sm": "sm",
    "md": "sm"
});
```